.github/workflows/main.yml

```diff
@@ -0,0 +1,72 @@
+name: Controller Action
```

Comment:
Thanks for the PR. But I don't want to change our workflows in this way.

We can update our conditions from:

```yaml
github.event_name == 'push' || !github.event.pull_request.head.repo.fork
``` 

to 

```yaml
(github.event_name == 'push' && github.repository == 'apache/incubator-opendal' ) || !github.event.pull_request.head.repo.fork
```